osgp/shared/osgp-kafka-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/kafka/da/signature/MessageSigner.java

```diff
@@ -345,6 +381,8 @@ public static Builder newBuilder() {
 
         private boolean isSigningEnabled;
 
+        private boolean isStripHeaders;
```

Comment:
I thought that our convention was to use the is prefix only for getters, not for fields?